### DISCUSSION TITLE
Ollama Integration?

### DISCUSSION THREAD

**Discussion by ricklangiii** [Budibase/budibase#14119] [Help] [answered]
I think I can hack together an automation with n8n, but it sure would be awesome to have this built into Budibase like the OpenAI/ChatGPT functionality.  Any idea where this is on the roadmap, or elegant workarounds to achieve the functionality?

**Comment by ConorWebb96** [comment, marked as answer]
Hey @ricklangiii,

This isn't currently isn't on the road map. This could be added to the core project, created a feature request [here](https://github.com/Budibase/budibase/issues/14126) for this.
